### ISSUE TITLE
Provide repo description

### ISSUE THREAD

**Issue by anselmbradford** [USDepartmentofLabor/Child-Labor#70]
Please add a short summary description of what this project at the top of this repository so that it shows up in third-party websites like https://code.gov/#/explore-code/repos/37860521

**Comment by aquilent-dgemoets** [comment]
Okay...how?

**Comment by anselmbradford** [comment]
@aquilent-dgemoets http://stackoverflow.com/questions/7757751/how-do-you-change-a-repository-description-on-github If you don't see an `Edit` text button, an owner of this repository either needs to add you or they need to update it directly.

**Comment by anselmbradford** [comment]
Actually I see you have committed to the repository already, so you should be able to click in the `No description or website provided.` text on the main repository page at https://github.com/USDepartmentofLabor/Child-Labor or click the `Edit` text button if that is available (I forget if that only appears after some initial text is added).

**Comment by aquilent-dgemoets** [comment]
Yes I saw that same SO entry.  There was no edit button and I uploaded a file named description and it didn't work.  I guess we'll just try to figure it out.  Nothing is just simple and intuitive.

**Comment by aquilent-dgemoets** [comment]
According to [this article](http://stackoverflow.com/questions/15406274/changing-repository-description-in-git), you can only change the GitHub repository description of a project you **own**. Simple read/write access is not enough to change that.  So this will have to be performed by DOL.

**Comment by mkotb** [comment]
An addition of a README or something would be nice if the DOL doesn't appear to be willing to set the description anytime soon :P

**Comment by jfc3-dol** [comment]
This must have been done because the application has been deployed a few times since this issue was created.

**Comment by anselmbradford** [comment]
Nope, still not done:

<img width="401" alt="screen shot 2018-09-12 at 11 35 30 am" src="https://user-images.githubusercontent.com/704760/45436174-29505600-b680-11e8-8261-5088669946ac.png">

It looks like code.gov changed how they pull in the description and this repo isn't even [listed there](https://code.gov/#!/browse-projects?agencies=DOL), but this is a super simple fix for the owner of this repo. It's literally clicking the `No description text …` and entering a description for this project (likely "iOS version of the Child Labor app."). Who's the owner of this repo?

**Comment by jfc3-dol** [comment]
Okay, we can re-open the issue.

I assumed it had been done but not note/closed in GitHub.
